app/views/imports/_form_main.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="overflow-auto text-nowrap">
+  <%= turbo_frame_tag "imports-table" do %>
+    <table class="table table-striped shadow rounded">
+      <thead>
+        <tr class="table-secondary">
+          <th>Feed Url</th>
+          <th>Status</th>
+          <th>Episode Count</th>
+          <th>Import Date</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% if imports.present? %>
+          <% imports.each do |import| %>
+            <% if %w[complete failed].include?(import.status) %>
```

Comment:
should this conditional be extracted into a helper? basically the conditional is `PodcastImport.finished?`, to determine whether to keep polling or not (happens in the else branch)

also, i found out that this conditional can be buggy; if i somehow bork the podcast import but somehow the podcast import status doesn't update to `failed`, then a user can theoretically get stuck in the else branch with the poller. (this has happened once in development, though i don't know how to replicate it.)

Reply:
I think you should add a few enums/helpers to the podcast/episode import models.  Enums are the rails way!

```rb
# these have to be string-style enums, since the db has strings not ints
enum status: {created: "created", started: "started"...}, prefix: true

def finalized?
  status_complete? || status_failed?
end
```

Hrmm... per getting stuck in a non-final status, should probably keep an eye out for that.  May need to rescue something, so we can set the status to failed.
```